test/integration/patients/patient/sidebar.js

```diff
@@ -110,12 +113,8 @@ context('patient sidebar', function() {
               'workspaces',
               'groups', // deprecated version of 'workspaces'
               'divider',
-              'optionsWidget1',
-              'optionsWidget2',
-              'optionsWidget3',
-              'optionsWidget4',
-              'optionsWidget5',
-              'optionsWidget6',
+              'hbsOptionsWidget1',
+              'hbsOptionsWidget2',
```

Comment:
Maybe we can remove these two as well? And just have a generic set of `hbsWidget` widgets that covers things.

Reply:
Yes we can remove it entirely, we don't need to test that handlebars works for options either.

Reply:
Cool, I removed those.